ush/ufoeval/run_ufo_hofx_test.sh

```diff
@@ -9,18 +9,55 @@
 # - Produce an EVA YAML
 # - Run EVA
 #-------------------------------------------------------------
-#--------------- User modified options below -----------------
-obtype=$1
-cycle=${2:-2021080100}
+
+# ==============================================================================
+usage() {
+  set +x
+  echo
+  echo "Usage: $0 [-c cycle] [-x] [-s] [-h] instrument [cycle]"
```

Comment:
Is this usage statement correct? Why `[cycle]` at the end?

Reply:
This is not correct (there were many iterations!): `[cycle]` removed